src/main/webapp/app/modules/login/LoginFooter.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Text } from '@adobe/react-spectrum'
+
+import { CommonConstants } from '../../shared/util/common.constants';
+
+// TODO: Test Cases
+// TODO : year should come dynamically 
```

Comment:
@dev-fe-dx26 , Having year as dynamic will be problematic as your tests will not be deterministic. So as is will be fine!